scss/_list-group.scss

```diff
@@ -64,8 +70,7 @@
 
   &:active {
     color: var(--#{$prefix}list-group-action-active-color);
-    background-color: var(--#{$prefix}list-group-action-active-bg);
-    border-color: var(--#{$prefix}list-group-active-border-color); // Boosted mod
+    background: var(--#{$prefix}list-group-action-active-bg); // Boosted mod: bg over bg-color
```

Comment:
Not sure if the constrast is enough, double check with Vincent?

Reply:
You mean in dark mode ? 

The contrast on light mode is 15.46, so it should pass any test
The contrast on dark background is 5.74 so it passes almost evey test (apart the AAA for small texts)

Reply:
No sorry I was speaaking about the `#f16e00` active marker on the `#ddd` background.
CAA gave me a 2.2:1 ratio for UI graphical elements which is a AA criteria.

Reply:
Let's keep it for the a11y review ?

Reply:
Can this comment be resolved?

Reply:
Resolved